pkg/rancher-desktop/components/EngineSelector.vue

```diff
@@ -62,4 +63,12 @@ export default {
 .container-engine::v-deep label {
   color: var(--input-label);
 }
+.locked::v-deep .radio-container span.radio-custom {
+    opacity: 1;
+    &:not([aria-checked="true"]) {
+      opacity: 1;
+      background-color: var(--radio-locked-bg);
+      box-shadow: var(--radio-locked-shadow);
+    }
+}
```

Comment:
Since the radio button implementation isn't easily wrapped, I think it might make sense to create a global style and a special class for locked radio buttons in `pkg/rancher-desktop/assets/styles/rancher-desktop.scss`. This will at least cut down on the amount of duplicated styles required for a consumer of lockable radio buttons.